Source/Scene/ArcGisMapServerImageryProvider.js

```diff
@@ -605,7 +700,32 @@ define([
         //>>includeEnd('debug');
 
         var url = buildImageUrl(this, x, y, level);
-        return ImageryProvider.loadImage(this, url);
+        if (!defined(this._requestNewToken)) {
+            return ImageryProvider.loadImage(this, url);
+        } else {
+            let that = this;
```

Comment:
You need to stick to ES5 in Cesium.  `var` instead of `let`.

Reply:
Thanks.

06470fe74d45a3578ba6b872456df4f1d668fba3